x-pack/plugin/core/src/main/resources/synthetics-policy.json

```diff
@@ -0,0 +1,12 @@
+{
+  "phases": {
+    "hot": {
+      "actions": {
+        "rollover": {
+          "max_size": "50gb",
```

Comment:
@andrewvc I assume this also works for synthetics for now?

Reply:
TBH we haven't done enough benchmarking to know, but it seems reasonable